src/Geta.Optimizely.Sitemaps/ServiceCollectionExtensions.cs

```diff
@@ -42,6 +43,7 @@ public static IServiceCollection AddSitemaps(this IServiceCollection services)
             services.AddSingleton<ISitemapLoader, SitemapLoader>();
             services.AddSingleton<ISitemapRepository, SitemapRepository>();
             services.AddSingleton<IContentFilter, ContentFilter>();
+            services.AddSingleton<IUriAugmenterService, DefaultUriAugmenterService>();
```

Comment:
You should not register the default augment service here. Instead, set it as a default type in the Options class and then set it from there. See how it is implemented in ProductFeed.